### PR TITLE
Added HazMat suites to Tiberian Dawn

### PR DESCRIPTION
**Pull request by Mailaender**
Closes https://github.com/OpenRA/OpenRA/issues/5030.

### CODE REVIEW

**Comment by pchote** [comment]
Great!  This is something that has bugged me for ages.

File: mods/cnc/rules/defaults.yaml [hunk 1]
```diff
@@ -204,6 +204,11 @@
 		CargoType: Infantry
 	HiddenUnderFog:
 	PoisonedByTiberium:
+		UpgradeTypes: hazmatsuites
```
Comment (cjshmyr) [inline mods/cnc/rules/defaults.yaml]: typo, `suits` or `suit`

Comment (Mailaender) [inline mods/cnc/rules/defaults.yaml]: Hehe, fixed.

**Comment by pchote** [comment]
Works as advertised, so :+1:

A followup pr should add some kind of selection decoration to infantry to indicate that the upgrade is active.

**Comment by GraionDilach** [comment]
I made a pip.

https://copy.com/Ty6WXfxYzxT21nSS

**Comment by obrakmann** [comment]
Thanks @GraionDilach, I added them in https://github.com/Mailaender/OpenRA/pull/56

**Comment by Mailaender** [comment]
Awesome, thanks! Can you also make those for hospital healing?

**Comment by pchote** [comment]
I believe that RA has a medic icon we can use for that.

**Comment by Phrohdoh** [comment]
This is looking pretty sharp!

**Comment by GraionDilach** [comment]
>I believe that RA has a medic icon we can use for that.

Indeed it had one. I took it out, converted to TD palette and cropped the canvas.
https://copy.com/MDuTLsTOpJoRuTxo

**Comment by Mailaender** [comment]
Thanks. Submitted it in https://github.com/OpenRA/OpenRA/pull/10680.

**Comment by GraionDilach** [comment]
Come to think of it, this will be wrong, they will overlap with each other/ranks.

**Comment by Mailaender** [comment]
They work together with ranks although with those tiny infantry it is already quite edgy. I added a `Description` which can be used in future tooltips or #3788.

**Comment by pchote** [comment]
You could set it up with upgrades to render the bio glyph in a different position if the hospital upgrade is already applied.

**Comment by GraionDilach** [comment]
My suggestion would be to shift hazmat and hospital into topright corner - that worked well in YR (for hosp and machine shop) and with the disguise pip in our RA/TS mod.

**Comment by obrakmann** [comment]
Let's do the fine-tuning in the spin-off PR. For now, this works as advertised. :+1:

**Comment by obrakmann** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-%28bleed%29/_compare/63626aadca490f8425cfaf74be010f391239265f)

**Pull request merged**